docs/source/user_guide/installation/index.rst

```diff
@@ -4,42 +4,30 @@ Installation
 PyBaMM is available on GNU/Linux, MacOS and Windows.
 It can be installed using ``pip`` or ``conda``, or from source.
 
-.. tab:: GNU/Linux and Windows
+.. tab:: pip
 
-   .. tab:: pip
+   PyBaMM can be installed via pip from `PyPI <https://pypi.org/project/pybamm>`__.
 
-      PyBaMM can be installed via pip from `PyPI <https://pypi.org/project/pybamm>`__.
+   .. code:: bash
 
-      .. code:: bash
+      pip install pybamm
 
-         pip install pybamm
+.. tab:: conda
 
-   .. tab:: conda
+   PyBaMM is part of the `Anaconda <https://docs.continuum.io/anaconda/>`_ distribution and is available as a conda package through the conda-forge channel.
 
-      PyBaMM is part of the `Anaconda <https://docs.continuum.io/anaconda/>`_ distribution and is available as a conda package through the conda-forge channel.
+   The ``pybamm`` package on conda-forge installs PyBaMM with all the `required and optional dependencies <index.html#dependencies>`_ available on conda-forge.
+   At the moment, PyBaMM's conda-forge distribution does not include the `IDAKLUSolver <../../api/solvers/idaklu_solver.html#idaklu-solver>`_.
```

Comment:
Same as above – I do think we should unify these pages (I was doing this last year, but it fell out of my radar some while later).